pio-proc/src/lib.rs

```diff
@@ -378,17 +389,29 @@ fn to_codegen(
     .parse()
     .unwrap();
     let program_size = max_program_size;
+
+    // This makes sure the file is added to the list
+    // of tracked files, so a change of that file triggers
+    // a recompile. Should be replaced by
+    // `proc_macro::tracked_path::path` when it is stable.
+    let dummy_include = match file {
+        Some(file_path) => quote! {let _ = include_bytes!( #file_path );},
```

Comment:
This could also be:
```suggestion
        Some(file_path) => quote! {const _:&[u8] = include_bytes!( #file_path );},
```
Not sure which one is better.